packages/amplify-graphql-api-construct-tests/src/__tests__/backends/all-auth-modes/app.ts

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env node
+import 'source-map-support/register';
+import { App, Stack, Duration } from 'aws-cdk-lib';
+import { Role, ServicePrincipal } from 'aws-cdk-lib/aws-iam';
+import { UserPool, CfnIdentityPool } from 'aws-cdk-lib/aws-cognito';
+import { NodejsFunction } from 'aws-cdk-lib/aws-lambda-nodejs';
+// @ts-ignore
+import { AmplifyGraphqlApi, AmplifyGraphqlDefinition } from '@aws-amplify/graphql-api-construct';
+import * as path from 'path';
+
+const packageJson = require('../package.json');
+
+const app = new App();
+const stack = new Stack(app, packageJson.name.replace(/_/g, '-'), {
+  env: { region: process.env.CLI_REGION || 'us-west-2' },
+});
+
+const userPool = new UserPool(stack, 'TestPool', {});
+// const identityPoolId = new CfnIdentityPool(stack, 'TestIdentityPool', { allowUnauthenticatedIdentities: true }).attrId;
+const identityPoolId = 'us-west-2:54ee61c9-8d51-46aa-8898-33e8578d2f75'; // Random id, value not used in this test
```

Comment:
Note: this is temporary, and we're tracking getting this uncommented/reverted in our open tracking ticket.